addons/gsri_opex/functions/sub/fn_subDeployCRRC.sqf

```diff
@@ -0,0 +1,9 @@
+//params["_target", "_player", "_params"];
+params["_target"];
+private _crrc = "B_Boat_Transport_01_F" createVehicle [0,0,0];
+private _spawner = _target getVariable "GSRI_FREMM_associatedSpawner";
+_crrc setDir (getDir _spawner + 180);
+_crrc setPosWorld getPosWorld _spawner;
+
+//todo : handle sub's movement
```

Comment:
Track TODO as issues :)

Reply:
Not a TODO, only a forgotten comment, removing.